script.js

```diff
@@ -1,200 +1,70 @@
-// output is hidden by default
-document.getElementById('output').style.visibility = 'hidden';
+const inputEl = document.querySelector('#weightInput');
+const outputEl = document.querySelector('.output');
+const unitSelectorEl = document.querySelector('.unit-selector');
+const placeholderEl = inputEl.placeholder;

```

Comment:
is `inputEl.placeholder` a dom element? meaning `typeof inputEl === typeof inputEl.placeholder` ?
Would this mislead people?

Reply:
Thanks for pointing out! Gonna nuke the bugger.